### DISCUSSION TITLE
Budget averages per period

### DISCUSSION THREAD

**Discussion by ryjogo** [firefly-iii/firefly-iii#8742] [General]
I'm not sure if i'm missing this in the application, but under the standard budget reports, there isn't anywhere with an average over the selected period. i.e. if i choose a budget report for example Media/Streaming/TV over 12 months - i get 10000, it would be very helpful to display 10000/12 - 833.33, or even better average per category in the budget.

Perhaps there is support for custom reports?

**Comment by JC5** [comment]
There is no support yet for custom reports, but some users have connected Firefly III to Grafana, where you can build whatever you want!
